src/pages/community/foundation/local-initiative.mdx

```diff
@@ -0,0 +1,136 @@
+---
+sidebarTitle: GraphQL Local
+description:
+  The GraphQL Local initiative helps the GraphQL Foundation foster a culture of
+  community and innovation, contributing to the success, health, and
+  sustainability of our ecosystem.
+---
+
+import { Button } from "../../../app/conf/_components/button"
+
+# GraphQL Local Initiative
+
+The GraphQL Foundation's primary mission is to ensure the continued evolution of
+the GraphQL specification and reference implementations. As part of this
+mission, we strive to provide collaboration infrastructure that fosters a
+culture of community and innovation. Local get-togethers play a crucial role in
+this, bringing together developers, library maintainers, vendors, and
+enthusiasts to share knowledge and collaborate.
+
+To better support these meets and clarify the process, we are excited to
+announce the official **GraphQL Local** initiative. This initiative provides a
+framework for organizing local GraphQL user groups and offers support each approved
+group (hereafter, a "Local"). Here’s what you need to know to get involved:
+
+## Requirements
+
+Organizers of a Local must agree:
+
+- All official events and related channels will be governed by the
+  **[GraphQL Code of Conduct](https://graphql.org/codeofconduct/)**.
+- Official communications will comply with the
+  **[GraphQL Brand Guidelines](https://graphql.org/brand/)**.
+- The Local accepts **GraphQL TSC oversight**:
+  - At least one
+    [GraphQL Technical Steering Committee (TSC) member](https://github.com/graphql/graphql-wg/blob/main/GraphQL-TSC.md#tsc-members)
```

Comment:
I think we need this to just be you; I am a little concerned this might feel like a big hill if the Local person doesn't know a TSC member and may be afraid to ask someone they don't know to be an admin

Reply:
That's a good point, I hadn't considered that they might think they need to ask a TSC member directly. I've added parenthesis to indicate that _we_ will assign a TSC member to make this clearer. I was going to add that they could request a specific TSC member but figured that this is unnecessary - also you're right it'll probably be me for most of them.